recipe/meta.yaml

```diff
@@ -24,19 +21,35 @@ requirements:
 
 test:
   commands:
+    # binaries
     - mdb_copy -V                                    # [unix]
     - mdb_dump -V                                    # [unix]
     - mdb_load -V                                    # [unix]
     - mdb_stat -V                                    # [unix]
-    - test -f ${PREFIX}/lib/liblmdb.a                # [unix]
+
+    # shared:
     - test -f ${PREFIX}/lib/liblmdb.so               # [unix]
+    - if not exist %LIBRARY_BIN%\lmdb.dll exit 1     # [win]
+
+    # TODO: do we want to do have static libraries at all?
+    # static:
+    - test -f ${PREFIX}/lib/liblmdb.a                # [not win]
+
+    # headers:
+    - test -f ${PREFIX}/include/lmdb.h               # [unix]
+    - if not exist %LIBRARY_INC%\lmdb.h exit 1       # [win]
 
 about:
-  home: http://symas.com/mdb/
-  license: OpenLDAP Public
+  home: https://www.openldap.org/software/repo.html
+  license: OLDAP-2.8
   license_file: libraries/liblmdb/LICENSE
   summary: A high-performance embedded transactional key-value store database.
+  # careful: the gitlab repo has different branches for very
+  # different projects - relevant here are the ones called mdb.*;
+  # also: there's a mirror under https://github.com/LMDB/lmdb
+  dev_url: https://git.openldap.org/openldap/openldap/-/tree/mdb.master
```

Comment:
Should we just link to the mirror? It is pretty clear how to get to maintained repos for there and makes it a bit easier to see what is LMDB specific

Reply:
Fine by me.